README.md

```diff
@@ -16,6 +16,15 @@ Portions Copyright (c) 2012-2013 The Cryptonote developers.
 - Our [Vulnerability Response Process](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md) encourages responsible disclosure
 - We are also available via [HackerOne](https://hackerone.com/monero)
 
+## Research
+
+The [Monero Research Lab](https://src.getmonero.org/resources/research-lab/) coordinates Monero research as a project workgroup. We welcome collaboration and contributions from outside researchers! Because not all Lab work and publications are distributed as traditional preprints or articles, they may be easy to miss if you are conducting literature reviews for your own Monero research. You are encouraged to get in touch with our researchers if you have questions, wish to collaborate, or would like guidance to help avoid unnecessarily duplicating earlier or known work.
```

Comment:
Great addition, @SarangNoether - thanks for taking the time to write up a note to help grow the community.

The description in the first sentence, "project workgroup" doesn't convey a lot to me. What's the best description? Perhaps "The Monero Research Lab is an open forum where the community coordinates research into Monero cryptography, protocol, heuristic fungibility, and more"

Might not hurt to add a pointer to the MRL articles, which provide both a good summary and a good impression.
"... from outside researchers! Some key research results are published as non-paywalled [papers](https://lab.getmonero.org/), while others are implemented directly into the Monero codebase. Because not all ..."

Reply:
This statement should also convey that while everyone is welcome to join in and participate in MRL research, it is a serious operation with a lot of knowledge contained within. I suspect there is somewhat of a reputation among other researchers, likely from early in Monero's history, that we don't do serious research. Let's change that view.

The link that's provided has the same list of papers as the one you suggested.

Reply:
Good motivating statement. Sorry about the link duplication, I was thinking that one of them pointed to the GitHub (fixed below).

Could rearrange the first few sentences to group into [description of group, we do serious work published openly] then [we welcome outside research and wish to help you get up to speed]. e.g.

> The [Monero Research Lab](https://github.com/monero-project/research-lab) is an open forum where the community coordinates research into Monero cryptography, protocol, heuristic fungibility, and more! Some key research results are published as non-paywalled [papers](https://lab.getmonero.org/), while others are implemented directly into the Monero codebase. We welcome collaboration and contributions from outside researchers. Because not all Lab work and publications are distributed as traditional preprints or articles, they may...